plat/kvm/Config.uk

```diff
@@ -26,13 +29,28 @@ config KVM_BOOT_PROTO_LXBOOT
 	help
 		Linux 64-bit Boot Protocol
 
+config KVM_BOOT_EFI_STUB
+	bool "EFI stub"
+	depends on KVM_VMM_QEMU && !KVM_VMM_FIRECRACKER
+	select OPTIMIZE_PIE
+	select UKPLAT_ACPI
+	depends on !PLAT_LINUXU
+	help
+		Prepend a fake, architecture specific, PE header to the
```

Comment:
Too many implementation details here. This is unnecessary because:
* If the implementation changes this will rot
* It  is not the information the user is looking for at this point
A generic description of the feature would suffice.